cpp/pipelines/add_authority_external_ref.cc

```diff
@@ -144,6 +334,15 @@ int Main(int argc, char* argv[]) {
         ParseDataDnbFile(marc_output_filename_or_dnb_input, mapping_txt_filename);
         return EXIT_SUCCESS;
     }
+    if (marc_input_filename_or_create_flag == "--generate_gnd_to_wiki") {
```

Comment:
Please just replace the existing parameter `--create_mapping_file` and overwrite the logic there instead of adding a new one